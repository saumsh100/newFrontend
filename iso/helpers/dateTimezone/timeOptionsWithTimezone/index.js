
import moment from 'moment-timezone';

/**
 * We store time and results in cache so we dont have to trigger this process all over again.
 */
let cachedTimezone = null;
let cachedResultsList = null;

/**
 * Creates a list of options for time selectors with values that includes timezone.
 * @param timezone
 * @return {Array}
 */
export default function (timezone = 'America/Vancouver') {
  if (typeof timezone !== 'string') {
    throw new Error('Timezone has to be string');
  }

  if (timezone === cachedTimezone && cachedResultsList) {
    return cachedResultsList;
  }

  const timeOptions = [];
  const totalHours = 24;
  const increment = 5;
  const increments = 60 / increment;

  let i;
  for (i = 0; i < totalHours; i += 1) {
    let j;
    for (j = 0; j < increments; j += 1) {
      const time = moment.tz(`1970-1-31 ${i}:${j * increment}`, 'YYYY-M-D H:mm', timezone);
      const value = time.format();
      const label = time.format('LT');
      timeOptions.push({
        value,
        label,
      });
    }
  }
  cachedTimezone = timezone;
  cachedResultsList = timeOptions;
  return timeOptions;
}

export {
  cachedResultsList,
  cachedTimezone,
};
