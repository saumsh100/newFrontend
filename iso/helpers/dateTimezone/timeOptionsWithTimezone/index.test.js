
import timeOptionsWithTimezone, { cachedTimezone, cachedResultsList } from './';

const timezone = 'America/Vancouver';

let results = null;

describe('timeOptionsWithTimezone', () => {
  it('generate options by including timezone', () => {
    results = timeOptionsWithTimezone(timezone);
    expect(results).toMatchSnapshot();
    expect(cachedTimezone).toEqual(timezone);
    expect(cachedResultsList).toEqual(results);
  });

  it('returns cached result', () => {
    const values = timeOptionsWithTimezone(timezone);
    expect(values).toEqual(results);
  });

  it('overwrites cached values when different timezone is provided', () => {
    const nyTimezone = 'America/New_York';
    const values = timeOptionsWithTimezone(nyTimezone);
    expect(values).not.toEqual(results);
    expect(cachedTimezone).toEqual(nyTimezone);
    expect(values).toEqual(cachedResultsList);
  });

  it('throws if timezone is not a string', () => {
    expect(() => {
      timeOptionsWithTimezone(123);
    }).toThrowError('Timezone has to be string');
  });
});
