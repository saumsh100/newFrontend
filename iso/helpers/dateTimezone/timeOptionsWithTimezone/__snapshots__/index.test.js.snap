// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timeOptionsWithTimezone generate options by including timezone 1`] = `
Array [
  Object {
    "label": "12:00 AM",
    "value": "1970-01-31T00:00:00-08:00",
  },
  Object {
    "label": "12:05 AM",
    "value": "1970-01-31T00:05:00-08:00",
  },
  Object {
    "label": "12:10 AM",
    "value": "1970-01-31T00:10:00-08:00",
  },
  Object {
    "label": "12:15 AM",
    "value": "1970-01-31T00:15:00-08:00",
  },
  Object {
    "label": "12:20 AM",
    "value": "1970-01-31T00:20:00-08:00",
  },
  Object {
    "label": "12:25 AM",
    "value": "1970-01-31T00:25:00-08:00",
  },
  Object {
    "label": "12:30 AM",
    "value": "1970-01-31T00:30:00-08:00",
  },
  Object {
    "label": "12:35 AM",
    "value": "1970-01-31T00:35:00-08:00",
  },
  Object {
    "label": "12:40 AM",
    "value": "1970-01-31T00:40:00-08:00",
  },
  Object {
    "label": "12:45 AM",
    "value": "1970-01-31T00:45:00-08:00",
  },
  Object {
    "label": "12:50 AM",
    "value": "1970-01-31T00:50:00-08:00",
  },
  Object {
    "label": "12:55 AM",
    "value": "1970-01-31T00:55:00-08:00",
  },
  Object {
    "label": "1:00 AM",
    "value": "1970-01-31T01:00:00-08:00",
  },
  Object {
    "label": "1:05 AM",
    "value": "1970-01-31T01:05:00-08:00",
  },
  Object {
    "label": "1:10 AM",
    "value": "1970-01-31T01:10:00-08:00",
  },
  Object {
    "label": "1:15 AM",
    "value": "1970-01-31T01:15:00-08:00",
  },
  Object {
    "label": "1:20 AM",
    "value": "1970-01-31T01:20:00-08:00",
  },
  Object {
    "label": "1:25 AM",
    "value": "1970-01-31T01:25:00-08:00",
  },
  Object {
    "label": "1:30 AM",
    "value": "1970-01-31T01:30:00-08:00",
  },
  Object {
    "label": "1:35 AM",
    "value": "1970-01-31T01:35:00-08:00",
  },
  Object {
    "label": "1:40 AM",
    "value": "1970-01-31T01:40:00-08:00",
  },
  Object {
    "label": "1:45 AM",
    "value": "1970-01-31T01:45:00-08:00",
  },
  Object {
    "label": "1:50 AM",
    "value": "1970-01-31T01:50:00-08:00",
  },
  Object {
    "label": "1:55 AM",
    "value": "1970-01-31T01:55:00-08:00",
  },
  Object {
    "label": "2:00 AM",
    "value": "1970-01-31T02:00:00-08:00",
  },
  Object {
    "label": "2:05 AM",
    "value": "1970-01-31T02:05:00-08:00",
  },
  Object {
    "label": "2:10 AM",
    "value": "1970-01-31T02:10:00-08:00",
  },
  Object {
    "label": "2:15 AM",
    "value": "1970-01-31T02:15:00-08:00",
  },
  Object {
    "label": "2:20 AM",
    "value": "1970-01-31T02:20:00-08:00",
  },
  Object {
    "label": "2:25 AM",
    "value": "1970-01-31T02:25:00-08:00",
  },
  Object {
    "label": "2:30 AM",
    "value": "1970-01-31T02:30:00-08:00",
  },
  Object {
    "label": "2:35 AM",
    "value": "1970-01-31T02:35:00-08:00",
  },
  Object {
    "label": "2:40 AM",
    "value": "1970-01-31T02:40:00-08:00",
  },
  Object {
    "label": "2:45 AM",
    "value": "1970-01-31T02:45:00-08:00",
  },
  Object {
    "label": "2:50 AM",
    "value": "1970-01-31T02:50:00-08:00",
  },
  Object {
    "label": "2:55 AM",
    "value": "1970-01-31T02:55:00-08:00",
  },
  Object {
    "label": "3:00 AM",
    "value": "1970-01-31T03:00:00-08:00",
  },
  Object {
    "label": "3:05 AM",
    "value": "1970-01-31T03:05:00-08:00",
  },
  Object {
    "label": "3:10 AM",
    "value": "1970-01-31T03:10:00-08:00",
  },
  Object {
    "label": "3:15 AM",
    "value": "1970-01-31T03:15:00-08:00",
  },
  Object {
    "label": "3:20 AM",
    "value": "1970-01-31T03:20:00-08:00",
  },
  Object {
    "label": "3:25 AM",
    "value": "1970-01-31T03:25:00-08:00",
  },
  Object {
    "label": "3:30 AM",
    "value": "1970-01-31T03:30:00-08:00",
  },
  Object {
    "label": "3:35 AM",
    "value": "1970-01-31T03:35:00-08:00",
  },
  Object {
    "label": "3:40 AM",
    "value": "1970-01-31T03:40:00-08:00",
  },
  Object {
    "label": "3:45 AM",
    "value": "1970-01-31T03:45:00-08:00",
  },
  Object {
    "label": "3:50 AM",
    "value": "1970-01-31T03:50:00-08:00",
  },
  Object {
    "label": "3:55 AM",
    "value": "1970-01-31T03:55:00-08:00",
  },
  Object {
    "label": "4:00 AM",
    "value": "1970-01-31T04:00:00-08:00",
  },
  Object {
    "label": "4:05 AM",
    "value": "1970-01-31T04:05:00-08:00",
  },
  Object {
    "label": "4:10 AM",
    "value": "1970-01-31T04:10:00-08:00",
  },
  Object {
    "label": "4:15 AM",
    "value": "1970-01-31T04:15:00-08:00",
  },
  Object {
    "label": "4:20 AM",
    "value": "1970-01-31T04:20:00-08:00",
  },
  Object {
    "label": "4:25 AM",
    "value": "1970-01-31T04:25:00-08:00",
  },
  Object {
    "label": "4:30 AM",
    "value": "1970-01-31T04:30:00-08:00",
  },
  Object {
    "label": "4:35 AM",
    "value": "1970-01-31T04:35:00-08:00",
  },
  Object {
    "label": "4:40 AM",
    "value": "1970-01-31T04:40:00-08:00",
  },
  Object {
    "label": "4:45 AM",
    "value": "1970-01-31T04:45:00-08:00",
  },
  Object {
    "label": "4:50 AM",
    "value": "1970-01-31T04:50:00-08:00",
  },
  Object {
    "label": "4:55 AM",
    "value": "1970-01-31T04:55:00-08:00",
  },
  Object {
    "label": "5:00 AM",
    "value": "1970-01-31T05:00:00-08:00",
  },
  Object {
    "label": "5:05 AM",
    "value": "1970-01-31T05:05:00-08:00",
  },
  Object {
    "label": "5:10 AM",
    "value": "1970-01-31T05:10:00-08:00",
  },
  Object {
    "label": "5:15 AM",
    "value": "1970-01-31T05:15:00-08:00",
  },
  Object {
    "label": "5:20 AM",
    "value": "1970-01-31T05:20:00-08:00",
  },
  Object {
    "label": "5:25 AM",
    "value": "1970-01-31T05:25:00-08:00",
  },
  Object {
    "label": "5:30 AM",
    "value": "1970-01-31T05:30:00-08:00",
  },
  Object {
    "label": "5:35 AM",
    "value": "1970-01-31T05:35:00-08:00",
  },
  Object {
    "label": "5:40 AM",
    "value": "1970-01-31T05:40:00-08:00",
  },
  Object {
    "label": "5:45 AM",
    "value": "1970-01-31T05:45:00-08:00",
  },
  Object {
    "label": "5:50 AM",
    "value": "1970-01-31T05:50:00-08:00",
  },
  Object {
    "label": "5:55 AM",
    "value": "1970-01-31T05:55:00-08:00",
  },
  Object {
    "label": "6:00 AM",
    "value": "1970-01-31T06:00:00-08:00",
  },
  Object {
    "label": "6:05 AM",
    "value": "1970-01-31T06:05:00-08:00",
  },
  Object {
    "label": "6:10 AM",
    "value": "1970-01-31T06:10:00-08:00",
  },
  Object {
    "label": "6:15 AM",
    "value": "1970-01-31T06:15:00-08:00",
  },
  Object {
    "label": "6:20 AM",
    "value": "1970-01-31T06:20:00-08:00",
  },
  Object {
    "label": "6:25 AM",
    "value": "1970-01-31T06:25:00-08:00",
  },
  Object {
    "label": "6:30 AM",
    "value": "1970-01-31T06:30:00-08:00",
  },
  Object {
    "label": "6:35 AM",
    "value": "1970-01-31T06:35:00-08:00",
  },
  Object {
    "label": "6:40 AM",
    "value": "1970-01-31T06:40:00-08:00",
  },
  Object {
    "label": "6:45 AM",
    "value": "1970-01-31T06:45:00-08:00",
  },
  Object {
    "label": "6:50 AM",
    "value": "1970-01-31T06:50:00-08:00",
  },
  Object {
    "label": "6:55 AM",
    "value": "1970-01-31T06:55:00-08:00",
  },
  Object {
    "label": "7:00 AM",
    "value": "1970-01-31T07:00:00-08:00",
  },
  Object {
    "label": "7:05 AM",
    "value": "1970-01-31T07:05:00-08:00",
  },
  Object {
    "label": "7:10 AM",
    "value": "1970-01-31T07:10:00-08:00",
  },
  Object {
    "label": "7:15 AM",
    "value": "1970-01-31T07:15:00-08:00",
  },
  Object {
    "label": "7:20 AM",
    "value": "1970-01-31T07:20:00-08:00",
  },
  Object {
    "label": "7:25 AM",
    "value": "1970-01-31T07:25:00-08:00",
  },
  Object {
    "label": "7:30 AM",
    "value": "1970-01-31T07:30:00-08:00",
  },
  Object {
    "label": "7:35 AM",
    "value": "1970-01-31T07:35:00-08:00",
  },
  Object {
    "label": "7:40 AM",
    "value": "1970-01-31T07:40:00-08:00",
  },
  Object {
    "label": "7:45 AM",
    "value": "1970-01-31T07:45:00-08:00",
  },
  Object {
    "label": "7:50 AM",
    "value": "1970-01-31T07:50:00-08:00",
  },
  Object {
    "label": "7:55 AM",
    "value": "1970-01-31T07:55:00-08:00",
  },
  Object {
    "label": "8:00 AM",
    "value": "1970-01-31T08:00:00-08:00",
  },
  Object {
    "label": "8:05 AM",
    "value": "1970-01-31T08:05:00-08:00",
  },
  Object {
    "label": "8:10 AM",
    "value": "1970-01-31T08:10:00-08:00",
  },
  Object {
    "label": "8:15 AM",
    "value": "1970-01-31T08:15:00-08:00",
  },
  Object {
    "label": "8:20 AM",
    "value": "1970-01-31T08:20:00-08:00",
  },
  Object {
    "label": "8:25 AM",
    "value": "1970-01-31T08:25:00-08:00",
  },
  Object {
    "label": "8:30 AM",
    "value": "1970-01-31T08:30:00-08:00",
  },
  Object {
    "label": "8:35 AM",
    "value": "1970-01-31T08:35:00-08:00",
  },
  Object {
    "label": "8:40 AM",
    "value": "1970-01-31T08:40:00-08:00",
  },
  Object {
    "label": "8:45 AM",
    "value": "1970-01-31T08:45:00-08:00",
  },
  Object {
    "label": "8:50 AM",
    "value": "1970-01-31T08:50:00-08:00",
  },
  Object {
    "label": "8:55 AM",
    "value": "1970-01-31T08:55:00-08:00",
  },
  Object {
    "label": "9:00 AM",
    "value": "1970-01-31T09:00:00-08:00",
  },
  Object {
    "label": "9:05 AM",
    "value": "1970-01-31T09:05:00-08:00",
  },
  Object {
    "label": "9:10 AM",
    "value": "1970-01-31T09:10:00-08:00",
  },
  Object {
    "label": "9:15 AM",
    "value": "1970-01-31T09:15:00-08:00",
  },
  Object {
    "label": "9:20 AM",
    "value": "1970-01-31T09:20:00-08:00",
  },
  Object {
    "label": "9:25 AM",
    "value": "1970-01-31T09:25:00-08:00",
  },
  Object {
    "label": "9:30 AM",
    "value": "1970-01-31T09:30:00-08:00",
  },
  Object {
    "label": "9:35 AM",
    "value": "1970-01-31T09:35:00-08:00",
  },
  Object {
    "label": "9:40 AM",
    "value": "1970-01-31T09:40:00-08:00",
  },
  Object {
    "label": "9:45 AM",
    "value": "1970-01-31T09:45:00-08:00",
  },
  Object {
    "label": "9:50 AM",
    "value": "1970-01-31T09:50:00-08:00",
  },
  Object {
    "label": "9:55 AM",
    "value": "1970-01-31T09:55:00-08:00",
  },
  Object {
    "label": "10:00 AM",
    "value": "1970-01-31T10:00:00-08:00",
  },
  Object {
    "label": "10:05 AM",
    "value": "1970-01-31T10:05:00-08:00",
  },
  Object {
    "label": "10:10 AM",
    "value": "1970-01-31T10:10:00-08:00",
  },
  Object {
    "label": "10:15 AM",
    "value": "1970-01-31T10:15:00-08:00",
  },
  Object {
    "label": "10:20 AM",
    "value": "1970-01-31T10:20:00-08:00",
  },
  Object {
    "label": "10:25 AM",
    "value": "1970-01-31T10:25:00-08:00",
  },
  Object {
    "label": "10:30 AM",
    "value": "1970-01-31T10:30:00-08:00",
  },
  Object {
    "label": "10:35 AM",
    "value": "1970-01-31T10:35:00-08:00",
  },
  Object {
    "label": "10:40 AM",
    "value": "1970-01-31T10:40:00-08:00",
  },
  Object {
    "label": "10:45 AM",
    "value": "1970-01-31T10:45:00-08:00",
  },
  Object {
    "label": "10:50 AM",
    "value": "1970-01-31T10:50:00-08:00",
  },
  Object {
    "label": "10:55 AM",
    "value": "1970-01-31T10:55:00-08:00",
  },
  Object {
    "label": "11:00 AM",
    "value": "1970-01-31T11:00:00-08:00",
  },
  Object {
    "label": "11:05 AM",
    "value": "1970-01-31T11:05:00-08:00",
  },
  Object {
    "label": "11:10 AM",
    "value": "1970-01-31T11:10:00-08:00",
  },
  Object {
    "label": "11:15 AM",
    "value": "1970-01-31T11:15:00-08:00",
  },
  Object {
    "label": "11:20 AM",
    "value": "1970-01-31T11:20:00-08:00",
  },
  Object {
    "label": "11:25 AM",
    "value": "1970-01-31T11:25:00-08:00",
  },
  Object {
    "label": "11:30 AM",
    "value": "1970-01-31T11:30:00-08:00",
  },
  Object {
    "label": "11:35 AM",
    "value": "1970-01-31T11:35:00-08:00",
  },
  Object {
    "label": "11:40 AM",
    "value": "1970-01-31T11:40:00-08:00",
  },
  Object {
    "label": "11:45 AM",
    "value": "1970-01-31T11:45:00-08:00",
  },
  Object {
    "label": "11:50 AM",
    "value": "1970-01-31T11:50:00-08:00",
  },
  Object {
    "label": "11:55 AM",
    "value": "1970-01-31T11:55:00-08:00",
  },
  Object {
    "label": "12:00 PM",
    "value": "1970-01-31T12:00:00-08:00",
  },
  Object {
    "label": "12:05 PM",
    "value": "1970-01-31T12:05:00-08:00",
  },
  Object {
    "label": "12:10 PM",
    "value": "1970-01-31T12:10:00-08:00",
  },
  Object {
    "label": "12:15 PM",
    "value": "1970-01-31T12:15:00-08:00",
  },
  Object {
    "label": "12:20 PM",
    "value": "1970-01-31T12:20:00-08:00",
  },
  Object {
    "label": "12:25 PM",
    "value": "1970-01-31T12:25:00-08:00",
  },
  Object {
    "label": "12:30 PM",
    "value": "1970-01-31T12:30:00-08:00",
  },
  Object {
    "label": "12:35 PM",
    "value": "1970-01-31T12:35:00-08:00",
  },
  Object {
    "label": "12:40 PM",
    "value": "1970-01-31T12:40:00-08:00",
  },
  Object {
    "label": "12:45 PM",
    "value": "1970-01-31T12:45:00-08:00",
  },
  Object {
    "label": "12:50 PM",
    "value": "1970-01-31T12:50:00-08:00",
  },
  Object {
    "label": "12:55 PM",
    "value": "1970-01-31T12:55:00-08:00",
  },
  Object {
    "label": "1:00 PM",
    "value": "1970-01-31T13:00:00-08:00",
  },
  Object {
    "label": "1:05 PM",
    "value": "1970-01-31T13:05:00-08:00",
  },
  Object {
    "label": "1:10 PM",
    "value": "1970-01-31T13:10:00-08:00",
  },
  Object {
    "label": "1:15 PM",
    "value": "1970-01-31T13:15:00-08:00",
  },
  Object {
    "label": "1:20 PM",
    "value": "1970-01-31T13:20:00-08:00",
  },
  Object {
    "label": "1:25 PM",
    "value": "1970-01-31T13:25:00-08:00",
  },
  Object {
    "label": "1:30 PM",
    "value": "1970-01-31T13:30:00-08:00",
  },
  Object {
    "label": "1:35 PM",
    "value": "1970-01-31T13:35:00-08:00",
  },
  Object {
    "label": "1:40 PM",
    "value": "1970-01-31T13:40:00-08:00",
  },
  Object {
    "label": "1:45 PM",
    "value": "1970-01-31T13:45:00-08:00",
  },
  Object {
    "label": "1:50 PM",
    "value": "1970-01-31T13:50:00-08:00",
  },
  Object {
    "label": "1:55 PM",
    "value": "1970-01-31T13:55:00-08:00",
  },
  Object {
    "label": "2:00 PM",
    "value": "1970-01-31T14:00:00-08:00",
  },
  Object {
    "label": "2:05 PM",
    "value": "1970-01-31T14:05:00-08:00",
  },
  Object {
    "label": "2:10 PM",
    "value": "1970-01-31T14:10:00-08:00",
  },
  Object {
    "label": "2:15 PM",
    "value": "1970-01-31T14:15:00-08:00",
  },
  Object {
    "label": "2:20 PM",
    "value": "1970-01-31T14:20:00-08:00",
  },
  Object {
    "label": "2:25 PM",
    "value": "1970-01-31T14:25:00-08:00",
  },
  Object {
    "label": "2:30 PM",
    "value": "1970-01-31T14:30:00-08:00",
  },
  Object {
    "label": "2:35 PM",
    "value": "1970-01-31T14:35:00-08:00",
  },
  Object {
    "label": "2:40 PM",
    "value": "1970-01-31T14:40:00-08:00",
  },
  Object {
    "label": "2:45 PM",
    "value": "1970-01-31T14:45:00-08:00",
  },
  Object {
    "label": "2:50 PM",
    "value": "1970-01-31T14:50:00-08:00",
  },
  Object {
    "label": "2:55 PM",
    "value": "1970-01-31T14:55:00-08:00",
  },
  Object {
    "label": "3:00 PM",
    "value": "1970-01-31T15:00:00-08:00",
  },
  Object {
    "label": "3:05 PM",
    "value": "1970-01-31T15:05:00-08:00",
  },
  Object {
    "label": "3:10 PM",
    "value": "1970-01-31T15:10:00-08:00",
  },
  Object {
    "label": "3:15 PM",
    "value": "1970-01-31T15:15:00-08:00",
  },
  Object {
    "label": "3:20 PM",
    "value": "1970-01-31T15:20:00-08:00",
  },
  Object {
    "label": "3:25 PM",
    "value": "1970-01-31T15:25:00-08:00",
  },
  Object {
    "label": "3:30 PM",
    "value": "1970-01-31T15:30:00-08:00",
  },
  Object {
    "label": "3:35 PM",
    "value": "1970-01-31T15:35:00-08:00",
  },
  Object {
    "label": "3:40 PM",
    "value": "1970-01-31T15:40:00-08:00",
  },
  Object {
    "label": "3:45 PM",
    "value": "1970-01-31T15:45:00-08:00",
  },
  Object {
    "label": "3:50 PM",
    "value": "1970-01-31T15:50:00-08:00",
  },
  Object {
    "label": "3:55 PM",
    "value": "1970-01-31T15:55:00-08:00",
  },
  Object {
    "label": "4:00 PM",
    "value": "1970-01-31T16:00:00-08:00",
  },
  Object {
    "label": "4:05 PM",
    "value": "1970-01-31T16:05:00-08:00",
  },
  Object {
    "label": "4:10 PM",
    "value": "1970-01-31T16:10:00-08:00",
  },
  Object {
    "label": "4:15 PM",
    "value": "1970-01-31T16:15:00-08:00",
  },
  Object {
    "label": "4:20 PM",
    "value": "1970-01-31T16:20:00-08:00",
  },
  Object {
    "label": "4:25 PM",
    "value": "1970-01-31T16:25:00-08:00",
  },
  Object {
    "label": "4:30 PM",
    "value": "1970-01-31T16:30:00-08:00",
  },
  Object {
    "label": "4:35 PM",
    "value": "1970-01-31T16:35:00-08:00",
  },
  Object {
    "label": "4:40 PM",
    "value": "1970-01-31T16:40:00-08:00",
  },
  Object {
    "label": "4:45 PM",
    "value": "1970-01-31T16:45:00-08:00",
  },
  Object {
    "label": "4:50 PM",
    "value": "1970-01-31T16:50:00-08:00",
  },
  Object {
    "label": "4:55 PM",
    "value": "1970-01-31T16:55:00-08:00",
  },
  Object {
    "label": "5:00 PM",
    "value": "1970-01-31T17:00:00-08:00",
  },
  Object {
    "label": "5:05 PM",
    "value": "1970-01-31T17:05:00-08:00",
  },
  Object {
    "label": "5:10 PM",
    "value": "1970-01-31T17:10:00-08:00",
  },
  Object {
    "label": "5:15 PM",
    "value": "1970-01-31T17:15:00-08:00",
  },
  Object {
    "label": "5:20 PM",
    "value": "1970-01-31T17:20:00-08:00",
  },
  Object {
    "label": "5:25 PM",
    "value": "1970-01-31T17:25:00-08:00",
  },
  Object {
    "label": "5:30 PM",
    "value": "1970-01-31T17:30:00-08:00",
  },
  Object {
    "label": "5:35 PM",
    "value": "1970-01-31T17:35:00-08:00",
  },
  Object {
    "label": "5:40 PM",
    "value": "1970-01-31T17:40:00-08:00",
  },
  Object {
    "label": "5:45 PM",
    "value": "1970-01-31T17:45:00-08:00",
  },
  Object {
    "label": "5:50 PM",
    "value": "1970-01-31T17:50:00-08:00",
  },
  Object {
    "label": "5:55 PM",
    "value": "1970-01-31T17:55:00-08:00",
  },
  Object {
    "label": "6:00 PM",
    "value": "1970-01-31T18:00:00-08:00",
  },
  Object {
    "label": "6:05 PM",
    "value": "1970-01-31T18:05:00-08:00",
  },
  Object {
    "label": "6:10 PM",
    "value": "1970-01-31T18:10:00-08:00",
  },
  Object {
    "label": "6:15 PM",
    "value": "1970-01-31T18:15:00-08:00",
  },
  Object {
    "label": "6:20 PM",
    "value": "1970-01-31T18:20:00-08:00",
  },
  Object {
    "label": "6:25 PM",
    "value": "1970-01-31T18:25:00-08:00",
  },
  Object {
    "label": "6:30 PM",
    "value": "1970-01-31T18:30:00-08:00",
  },
  Object {
    "label": "6:35 PM",
    "value": "1970-01-31T18:35:00-08:00",
  },
  Object {
    "label": "6:40 PM",
    "value": "1970-01-31T18:40:00-08:00",
  },
  Object {
    "label": "6:45 PM",
    "value": "1970-01-31T18:45:00-08:00",
  },
  Object {
    "label": "6:50 PM",
    "value": "1970-01-31T18:50:00-08:00",
  },
  Object {
    "label": "6:55 PM",
    "value": "1970-01-31T18:55:00-08:00",
  },
  Object {
    "label": "7:00 PM",
    "value": "1970-01-31T19:00:00-08:00",
  },
  Object {
    "label": "7:05 PM",
    "value": "1970-01-31T19:05:00-08:00",
  },
  Object {
    "label": "7:10 PM",
    "value": "1970-01-31T19:10:00-08:00",
  },
  Object {
    "label": "7:15 PM",
    "value": "1970-01-31T19:15:00-08:00",
  },
  Object {
    "label": "7:20 PM",
    "value": "1970-01-31T19:20:00-08:00",
  },
  Object {
    "label": "7:25 PM",
    "value": "1970-01-31T19:25:00-08:00",
  },
  Object {
    "label": "7:30 PM",
    "value": "1970-01-31T19:30:00-08:00",
  },
  Object {
    "label": "7:35 PM",
    "value": "1970-01-31T19:35:00-08:00",
  },
  Object {
    "label": "7:40 PM",
    "value": "1970-01-31T19:40:00-08:00",
  },
  Object {
    "label": "7:45 PM",
    "value": "1970-01-31T19:45:00-08:00",
  },
  Object {
    "label": "7:50 PM",
    "value": "1970-01-31T19:50:00-08:00",
  },
  Object {
    "label": "7:55 PM",
    "value": "1970-01-31T19:55:00-08:00",
  },
  Object {
    "label": "8:00 PM",
    "value": "1970-01-31T20:00:00-08:00",
  },
  Object {
    "label": "8:05 PM",
    "value": "1970-01-31T20:05:00-08:00",
  },
  Object {
    "label": "8:10 PM",
    "value": "1970-01-31T20:10:00-08:00",
  },
  Object {
    "label": "8:15 PM",
    "value": "1970-01-31T20:15:00-08:00",
  },
  Object {
    "label": "8:20 PM",
    "value": "1970-01-31T20:20:00-08:00",
  },
  Object {
    "label": "8:25 PM",
    "value": "1970-01-31T20:25:00-08:00",
  },
  Object {
    "label": "8:30 PM",
    "value": "1970-01-31T20:30:00-08:00",
  },
  Object {
    "label": "8:35 PM",
    "value": "1970-01-31T20:35:00-08:00",
  },
  Object {
    "label": "8:40 PM",
    "value": "1970-01-31T20:40:00-08:00",
  },
  Object {
    "label": "8:45 PM",
    "value": "1970-01-31T20:45:00-08:00",
  },
  Object {
    "label": "8:50 PM",
    "value": "1970-01-31T20:50:00-08:00",
  },
  Object {
    "label": "8:55 PM",
    "value": "1970-01-31T20:55:00-08:00",
  },
  Object {
    "label": "9:00 PM",
    "value": "1970-01-31T21:00:00-08:00",
  },
  Object {
    "label": "9:05 PM",
    "value": "1970-01-31T21:05:00-08:00",
  },
  Object {
    "label": "9:10 PM",
    "value": "1970-01-31T21:10:00-08:00",
  },
  Object {
    "label": "9:15 PM",
    "value": "1970-01-31T21:15:00-08:00",
  },
  Object {
    "label": "9:20 PM",
    "value": "1970-01-31T21:20:00-08:00",
  },
  Object {
    "label": "9:25 PM",
    "value": "1970-01-31T21:25:00-08:00",
  },
  Object {
    "label": "9:30 PM",
    "value": "1970-01-31T21:30:00-08:00",
  },
  Object {
    "label": "9:35 PM",
    "value": "1970-01-31T21:35:00-08:00",
  },
  Object {
    "label": "9:40 PM",
    "value": "1970-01-31T21:40:00-08:00",
  },
  Object {
    "label": "9:45 PM",
    "value": "1970-01-31T21:45:00-08:00",
  },
  Object {
    "label": "9:50 PM",
    "value": "1970-01-31T21:50:00-08:00",
  },
  Object {
    "label": "9:55 PM",
    "value": "1970-01-31T21:55:00-08:00",
  },
  Object {
    "label": "10:00 PM",
    "value": "1970-01-31T22:00:00-08:00",
  },
  Object {
    "label": "10:05 PM",
    "value": "1970-01-31T22:05:00-08:00",
  },
  Object {
    "label": "10:10 PM",
    "value": "1970-01-31T22:10:00-08:00",
  },
  Object {
    "label": "10:15 PM",
    "value": "1970-01-31T22:15:00-08:00",
  },
  Object {
    "label": "10:20 PM",
    "value": "1970-01-31T22:20:00-08:00",
  },
  Object {
    "label": "10:25 PM",
    "value": "1970-01-31T22:25:00-08:00",
  },
  Object {
    "label": "10:30 PM",
    "value": "1970-01-31T22:30:00-08:00",
  },
  Object {
    "label": "10:35 PM",
    "value": "1970-01-31T22:35:00-08:00",
  },
  Object {
    "label": "10:40 PM",
    "value": "1970-01-31T22:40:00-08:00",
  },
  Object {
    "label": "10:45 PM",
    "value": "1970-01-31T22:45:00-08:00",
  },
  Object {
    "label": "10:50 PM",
    "value": "1970-01-31T22:50:00-08:00",
  },
  Object {
    "label": "10:55 PM",
    "value": "1970-01-31T22:55:00-08:00",
  },
  Object {
    "label": "11:00 PM",
    "value": "1970-01-31T23:00:00-08:00",
  },
  Object {
    "label": "11:05 PM",
    "value": "1970-01-31T23:05:00-08:00",
  },
  Object {
    "label": "11:10 PM",
    "value": "1970-01-31T23:10:00-08:00",
  },
  Object {
    "label": "11:15 PM",
    "value": "1970-01-31T23:15:00-08:00",
  },
  Object {
    "label": "11:20 PM",
    "value": "1970-01-31T23:20:00-08:00",
  },
  Object {
    "label": "11:25 PM",
    "value": "1970-01-31T23:25:00-08:00",
  },
  Object {
    "label": "11:30 PM",
    "value": "1970-01-31T23:30:00-08:00",
  },
  Object {
    "label": "11:35 PM",
    "value": "1970-01-31T23:35:00-08:00",
  },
  Object {
    "label": "11:40 PM",
    "value": "1970-01-31T23:40:00-08:00",
  },
  Object {
    "label": "11:45 PM",
    "value": "1970-01-31T23:45:00-08:00",
  },
  Object {
    "label": "11:50 PM",
    "value": "1970-01-31T23:50:00-08:00",
  },
  Object {
    "label": "11:55 PM",
    "value": "1970-01-31T23:55:00-08:00",
  },
]
`;
